docs/concepts/CLQL.md

```diff
@@ -350,12 +350,15 @@ query:
 
 If run in the CLI with `lingo run rewrite` the user is prompted with a prompt like:
 ```
+Syncing your repo...
+Running rewrite flow...
 StringLiteral["Goodbye, World"]:
 ```
+Where the user can enter custom input to replace the string or have it replaced by the default we defined: "Goodbye, World".
```

Comment:
I think an extended shell example would be good. Or even better, a gif of the rewrite in action